File: app/src/main/java/com/gpacini/daysuntil/data/model/RealmEvent.java
package com.gpacini.daysuntil.data.model;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by gavinpacini on 10/10/15.
 */
public class RealmEvent extends RealmObject {

    @PrimaryKey
    private String uuid;

    private String title;

    private long timestamp;

    public String getUuid() {
        return uuid;
    }

    public void setUuid(String uuid) {
        this.uuid = uuid;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public long getTimestamp() {
        return timestamp;
    }

    public void setTimestamp(long timestamp) {
        this.timestamp = timestamp;
    }
}

File: app/src/main/java/com/gpacini/daysuntil/data/model/RealmEvent.kt
package com.gpacini.daysuntil.data.model

import io.realm.RealmObject
import io.realm.annotations.PrimaryKey

/**
 * Created by gavinpacini on 10/10/15.
 */
public open class RealmEvent : RealmObject() {

    @PrimaryKey
    public open var uuid: String? = null

    public open var title: String? = null

    public open var timestamp: Long = 0
}